src/Control/Monad/Freer/Fresh.hs

```diff
@@ -44,6 +44,10 @@ fresh :: Member Fresh effs => Eff effs Int
 fresh = send Fresh
 
 -- | Handler for 'Fresh' effects, with an 'Int' for a starting value.
-runFresh' :: Eff (Fresh ': effs) a -> Int -> Eff effs a
-runFresh' m s =
+runFresh :: Eff (Fresh ': effs) a -> Int -> Eff effs a
+runFresh m s =
```

Comment:
There is another PR (#25) with similar changes. I'll cherry pick your changes into it. Would you mind rebasing your commits on top of it afterwards?